### ISSUE TITLE
Change the Coco export to use the image file name / path instead of url

### ISSUE THREAD

**Issue by crubier** [labelflow/labelflow#176]
**Is your feature request related to a problem? Please describe.**

Right now our export (so the whole app) is pretty useless because the coco export references images URL that are only internal to the web app. So the users can't do anything of it.

**Describe the solution you'd like (import visuals when possible)**

- [ ]  Change the Coco export to use the image file name / path instead of url 

Cancelled (will be done in another issue):  Add an export option to export all images in a dataset in a zip file

**Describe alternatives you've considered**

**Additional context**

Blocked by #263
